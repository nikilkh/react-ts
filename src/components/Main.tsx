import { Children } from "react";
import { Route, Routes } from "react-router-dom";
import App from "../App";
// import App from "../App";
import BookingPage from "./BookingPage";
import { LandPage } from "./LandPage";
import MoviesContainer from "./MoviesContainer";


export default function Main() {
  return (
    <Routes>
      <Route path="/" element={<App ></App>}>
        <Route path=":movie" element={<MoviesContainer></MoviesContainer>}></Route>
        
      </Route>
      
      <Route path="/bookingpage" element={<BookingPage></BookingPage>}></Route>
    </Routes>
  );
}