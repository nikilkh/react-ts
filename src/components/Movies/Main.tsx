import StateContextProvider from "../context";
import { LandPage } from "./Landpage/LandPage";

export default function Main() {
  return (
    <>
      <StateContextProvider>
        <LandPage></LandPage>
      </StateContextProvider>
    </>
  );
}
