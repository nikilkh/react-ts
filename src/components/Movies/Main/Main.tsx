import StateContextProvider from "../../Context/context";
import { LandPage } from "../LandPage";

export const Main = () => {
  return (
    <>
      <StateContextProvider>
        <LandPage></LandPage>
      </StateContextProvider>
    </>
  );
}
