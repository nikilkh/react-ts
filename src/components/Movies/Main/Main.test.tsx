import { render } from "@testing-library/react";
import { Main } from "./Main";

describe("renders learn react link", () => {
  it("should render hello world", () => {
    const {container} = render(<Main/>);
    expect(container).toMatchSnapshot();
    
  });
});