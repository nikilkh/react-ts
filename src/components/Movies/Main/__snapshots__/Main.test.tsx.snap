// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders learn react link should render hello world 1`] = `
<div>
  <div
    class="sc-gsnTZi ToOMS"
  >
    <header
      class="sc-dkzDqf cDKfbt"
    >
      <h1>
        Movie List
      </h1>
    </header>
    <div
      class="sc-hKMtZM kBVPFl"
    >
      <input
        placeholder="search your movie here"
        value=""
      />
      <button
        class="sc-eCYdqJ jAibXW"
        data-testid="searchButton"
      >
        <img
          alt="search icon"
          src="searchIcon.svg"
        />
      </button>
    </div>
    <div
      class="sc-jSMfEi iAkGmp"
    >
      <h3>
        loading...
      </h3>
    </div>
  </div>
</div>
`;
